examples/translation/t5/evaluate_wmt.py

```diff
@@ -14,13 +14,13 @@ def chunks(lst, n):
         yield lst[i : i + n]
 
 
-def generate_translations(lns, output_file_path, batch_size, device):
+def generate_translations(lns, output_file_path, model_size, batch_size, device):
```

Comment:
`model_size` is a bit misleading, `model_name: str` preferable IMO

Reply:
yeah agree will change that when merging bart and t5 example